pyccel/codegen/printing/cwrappercode.py

```diff
@@ -240,6 +240,8 @@ def _print_PyArgKeywords(self, expr):
     def _print_PyModule_AddObject(self, expr):
         name = self._print(expr.name)
         var  = self._print(expr.variable)
+        if not expr.variable.dtype is PyccelPyObject():
```

Comment:
More Pythonic is writing
```suggestion
        if expr.variable.dtype is not PyccelPyObject():
```

Reply:
Huh, pylint used to raise an error about this